app/internal/comment.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+from typing import Dict, List
+
+from sqlalchemy.orm.session import Session
+
+from app.database.models import Comment, Event
+from app.internal import utils
+
+
+def create_comment(session: Session, event: Event, content: str) -> None:
+    """Creates a comment instance in the DB.
+
+    Args:
+        session (Session): DB session.
+        event (Event): Instance of the event to create a comment for.
+        content (str): The content of the comment.
+
+    Returns:
+        None
+
+    Raises:
+        None
+    """
+    data = {
+        'user': utils.get_current_user(session),
+        'event': event,
+        'content': content,
+        'time': datetime.datetime.now()
+    }
+    utils.create_model(session, Comment, **data)
+
+
+def parse_comment(session: Session, comment: Comment) -> Dict[str, str]:
+    """Returns a dictionary with the comment's info.
+
+    Args:
+        session (Session): DB session.
+        comment (Comment): Comment instance to parse.
+
+    Returns:
+        dict(str: str): Comment's info.
+                        'id' - The comment's.
+                        'avatar' - Commentor's profile image.
+                        'username' - Commentor's username.
+                        'time' - Comment's posting time.
+                        'content' - Comment's content
+
+    Raises:
+        None
+    """
+    user = utils.get_user(session, comment.user_id)
+    return {
+        'id': comment.id,
+        'avatar': user.avatar,
+        'username': user.username,
+        'time': comment.time.strftime(r'%d/%m/%Y %H:%M'),
+        'content': comment.content
+    }
+
+
+def display_comments(session: Session, event: Event) -> List[Dict[str, str]]:
```

Comment:
What do you mean by exporting it as a route?
This is a utility function used by path operations to get all necessary details of comments for displaying purposes.
The comment themselves are passed to a template and can be viewed by the user.
If your intention is to return the information as a JSON rather than a list it can be done.

Reply:
> If your intention is to return the information as a JSON rather than a list it can be done.

Yap, this will allow us to load it with JS and will make the access using API simpler :)